keras_nlp/tokenizers/sentence_piece_tokenizer.py

```diff
@@ -255,3 +256,8 @@ def detokenize(self, inputs):
         if unbatched:
             outputs = tf.squeeze(outputs, 0)
         return outputs
+
+    def compute_output_spec(self, input_spec) -> keras.KerasTensor:
```

Comment:
Ah I see why things are failing. Just remove the type annotation. We don't generally use them in Keras right now. Then the usage is inside the function itself (and the function is only called in Keras 3).